frontend/packages/ux-editor/src/utils/formLayoutUtils.ts

```diff
@@ -386,6 +386,15 @@ export const isComponentTypeValidChild = (
 export const getChildIds = (layout: IInternalLayout, parentId: string): string[] =>
   layout.order?.[parentId] || [];
 
+/**
+ * Recursively finds all the children of a container.
+ * @param layout The layout to search in.
+ * @param parentId The id of the container to find all children of.
+ * @returns An array of all the children of the container.
+ */
+export const getAllDescendants = (layout: IInternalLayout, parentId: string): string[] =>
+  getChildIds(layout, parentId).flatMap((id) => [id, ...getAllDescendants(layout, id)]);
```

Comment:
Fin funksjon! Her kan vi alternativt bruke `prepend` fra `arrayUtils` i `shared` (som for øvrig bør flyttes til `ArrayUtils` i `@studio/pure-functions`).

Reply:
Jeg endret denne til å bruke `prepend`, og flyttet `prepend` til `pure-functions` sin `ArrayUtils` siden den ikke ble brukt noe annet sted.